exonum/src/proto/schema/exonum/blockchain.proto

```diff
@@ -34,15 +34,15 @@ message Block {
 
 message TxLocation {
   uint64 block_height = 1;
-  uint64 position_in_block = 2;
+  uint32 position_in_block = 2;
 }
 
 // Location of an isolated call within a block.
 message CallInBlock {
   oneof call {
     // Call of a transaction within the block. The value is the zero-based
     // transaction index.
-    uint64 transaction = 1;
+    uint32 transaction = 1;
```

Comment:
Will we solve possible issues with field naming in a separate PR? AFAIK, @dmitry-timofeev is not quite satisfied with the present naming; I sort of agree. Would something like `transaction_index` / `service_before_transactions` / `service_after_transactions` be better?

Reply:
These changes was made by @dmitry-timofeev 's commit 😄 And he's changed only type of the field.

Reply:
Anyway, I think it would be better to change naming in another PR.